pkgs/by-name/py/pywal16/package.nix

```
@@ -3,33 +3,56 @@
   python3,
   fetchFromGitHub,
   imagemagick,
+  installShellFiles,
+  nix-update-script,
 }:
 
 python3.pkgs.buildPythonApplication rec {
   pname = "pywal16";
-  version = "3.6.0";
+  version = "3.7.2";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "eylles";
     repo = "pywal16";
     rev = "refs/tags/${version}";
```

```suggestion
    tag = version;
```
This is now possible and is preferred.